src/main/resources/vm/script/datamanagement/refresh.vm

```diff
@@ -54,7 +60,13 @@ function refresh_token {
 
         sleep 240
     done 
-    
+
+    ## check if the refreshing process loop is stopped
+    if [[ "$process_error" == true ]]; then
```

Comment:
Because of the pipe, the command in the loop run in a subshell. When you call exit, only the subshell is terminated, and not the parent process. 
testing ct-tiqua tooked enough time for me to notice this behavior, Test GME was fast i couldn't detect those bugs (refresh token and the exit)

Reply:
Hello Axel,
You were right, this line was unnecessary, the exit is enough. i think i was just over suspicious.
Thanks